### PR TITLE
Changing the build tab to describe Quick Deploy

### PR DESCRIPTION
**Pull request by heyaudace**
#### Added overview tab to provide overall description on how to proceed with building instant game with quick deploy.

screen shot 2018-08-28 at 11 57 58 am

### CODE REVIEW

**Comment by heyaudace** [comment]
Added overview tab according this morning's sync, and changed the UI screenshot

File: GooglePlayInstant/Editor/PlayInstantEditorMenu.cs [hunk 1]
```diff
@@ -69,30 +69,30 @@ private static void SetUpPlayInstantSdk()
             PlayInstantSdkInstaller.SetUp();
         }
 
-        [MenuItem("PlayInstant/Quick Deploy/AssetBundle Creation...", false, 300)]
+        [MenuItem("PlayInstant/Quick Deploy/Overview...", false, 300)]
+        private static void BuildApkSettings()
```
Comment (thanwah) [inline GooglePlayInstant/Editor/PlayInstantEditorMenu.cs]: Rename method

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs [hunk 2]
```diff
@@ -127,6 +127,38 @@ private static void UpdateGuiFocus(ToolBarSelectedButton currentTab)
             }
         }
 
+        private void OnGuiOverviewSelect()
+        {
+            var descriptionTextStyle = CreateDescriptionTextStyle();
+            EditorGUILayout.LabelField("Building Instant Game with Quick Deploy", EditorStyles.boldLabel);
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: "About Quick Deploy"

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs [hunk 3]
```diff
@@ -127,6 +127,38 @@ private static void UpdateGuiFocus(ToolBarSelectedButton currentTab)
...
+            EditorGUILayout.BeginVertical(UserInputGuiStyle);
+            EditorGUILayout.Space();
+            EditorGUILayout.Space();
+            EditorGUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField("Building an instant game with Quick Deploy will significantly reduce the size" +
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: `Quick Deploy can significantly reduce the size of a Unity-based instant app by packaging some assets in an AssetBundle that is retrieved from a server during app startup.`

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs [hunk 4]
```diff
@@ -127,6 +127,38 @@ private static void UpdateGuiFocus(ToolBarSelectedButton currentTab)
...
+                                       " of your game by storing the main AssetBundle of the game to a remote server.",
+                descriptionTextStyle);
+            EditorGUILayout.EndHorizontal();
+            EditorGUILayout.Space();
+            EditorGUILayout.Space();
+            EditorGUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField("Use Quick Deploy features to Build and Deploy AssetBundle, as well as to " +
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: `"Use the \"Bundle Creation\" tab to build an AssetBundle containing the game's main scene. Then upload the AssetBundle to Google Cloud Storage via the \"Bundle Deployment\" tab. Finally, use the \"Loading Screen\" tab to select an image to display on the loading screen and the URL that points to the uploaded AssetBundle."`

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs [hunk 5]
```diff
@@ -127,6 +127,38 @@ private static void UpdateGuiFocus(ToolBarSelectedButton currentTab)
...
+                                       "Create a Loading Screen that will be displayed at game startup when the " +
+                                       "AssetBundle will be downloaded from a remote server. ", descriptionTextStyle);
+            EditorGUILayout.EndHorizontal();
+            EditorGUILayout.Space();
+            EditorGUILayout.Space();
+            EditorGUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField("Use GooglePlayInstant's Build Settings Window to change " +
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: `"Use Google Play Instant's \"{0}\" window to customize the scenes included in the instant app. Then use the \"{1}\" menu option to test the instant app loading the AssetBundle from the remote server. Finally, select the \"{2}\" menu option to build the app in a manner suitable for publishing on Play Console."`

Reference existing strings for:
0 = Build Settings
1 = Build and Run
2 = Build for Play Console

File: GooglePlayInstant/Editor/PlayInstantEditorMenu.cs (hunk 1 above)
Comment (heyaudace) [reply GooglePlayInstant/Editor/PlayInstantEditorMenu.cs]: Resolved.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs (hunk 2 above)
Comment (heyaudace) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Resolved.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs (hunk 3 above)
Comment (heyaudace) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Resolved.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs (hunk 5 above)
Comment (heyaudace) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Resolved.

Wasn't sure what you meant by "Existing Strings" tho. Lmk if you had something else in mind than what i did.

**Review by heyaudace — COMMENTED** [review]
### Addressed all review comments.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs (hunk 3 above)
Comment (thanwah) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Unless I'm misreading something, this wasn't done.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs (hunk 4 above)
Comment (heyaudace) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Resolved.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs (hunk 3 above)
Comment (heyaudace) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Oops, had left out one commit by accident. Resolved now.

**Review by heyaudace — COMMENTED** [review]
### Resolved.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs [hunk 6]
```diff
@@ -123,6 +123,43 @@ private static void UpdateGuiFocus(ToolBarSelectedButton currentTab)
...
         }
 
+        private void OnGuiOverviewSelect()
+        {
+            var descriptionTextStyle = CreateDescriptionTextStyle();
+            EditorGUILayout.LabelField("About Quick Deploy", EditorStyles.boldLabel);
+            EditorGUILayout.BeginVertical(UserInputGuiStyle);
+            EditorGUILayout.Space();
+            EditorGUILayout.Space();
+            EditorGUILayout.BeginHorizontal();
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs:133]: You have a series of horizontal blocks on each section that only contains a single LabelField. If you remove these BeginHorizontal/EndHorizontal does anything change?

**Review by thanwah — APPROVED** [review]
LGTM with one question.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs (hunk 6 above)
Comment (heyaudace) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs:133]: @thanwah  This is what you'd get:

screen shot 2018-08-28 at 12 42 32 pm

**Pull request merged**